src/vmem.c

```diff
@@ -28,18 +28,25 @@
 #include "common.h"
 #include "plugin.h"
 
-#if KERNEL_LINUX
+#if KERNEL_DARWIN
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <mach/mach.h>
+#endif
+
+#if KERNEL_LINUX || KERNEL_DARWIN
 static const char *config_keys[] =
 {
   "Verbose"
 };
 static int config_keys_num = STATIC_ARRAY_SIZE (config_keys);
 
 static int verbose_output = 0;
-/* #endif KERNEL_LINUX */
+/* #endif KERNEL_LINUX || KERNEL_DARWIN */
```

Comment:
it is an existing comment above the #else to express that it is the end of the linux (or macosx now) define. After the else an error is set because of unsupported os.

So I added darwin to the commented line since it is now part of that define block.